examples/sdk/browser-keysigner/index.ts

```diff
@@ -0,0 +1,60 @@
+import { DfnsApiClient } from '@dfns/sdk'
```

Comment:
nit: given this is meant to be used in the browser, would a browser based example be more appropriate?